README.md

```diff
@@ -33,6 +33,81 @@ Python | macOS | Xcode | iPadOS, iOS |
 :------:|:------:|:------:|:------:|
 3.8 | 13.1 | 14.2 | 16.2 |
 
+## <a name="using-converted-weights"></a> Using Converted Weights from Hugging Face Hub
+
+<details>
+  <summary> Click to expand </summary>
+
+🤗 Hugging Face ran the [conversion procedure](#converting-models-to-coreml) on the following models and made the Core ML weights publicly available in the Hub:
```

Comment:
Could we mention the complementary nature of using ready-made models from the Hub versus converting custom ones? e.g. "If you would like to convert a version of Stable Diffusion that is not already available on the Hub, please refer to the [Converting Models to Core ML](#converting-models-to-core-ml)" section?